components/ratgdo/ratgdo.cpp

```diff
@@ -471,7 +460,12 @@ namespace ratgdo {
             return;
         }
 
-        this->door_action(DoorAction::CLOSE);
+        if(this->obstruction_sensor_detected_){
+			this->door_action(DoorAction::CLOSE);
+        }else if(*this->door_state == DoorState::OPEN){
+			ESP_LOGD(TAG, "No obstruction sensors detected. Close using TOGGLE.");
+            this->door_action(DoorAction::TOGGLE);
+        }
```

Comment:
`clang-format --style=Webkit -i  components/ratgdo/*.h components/ratgdo/*.cpp components/ratgdo/**/*.cpp components/ratgdo/**/*.h  `

Reply:
nevermind, I'll just realized the pre-commit hooks were not working so I fix that and it will be auto now